src/assets/scripts/aircraft/AircraftInstanceModel.js

```diff
@@ -904,12 +903,15 @@ export default class Aircraft {
 
         let isExpeditingString = '';
         if (expedite) {
-            isExpeditingString = 'and expedite';
+            isExpeditingString = ' and expedite';
```

Comment:
I understand. Trouble is that `isExpeditedString` may or may not contain a value. If it is an empty string, then the template string would have a trailing space... hence the addition of the space in the expedite string. If it's present, it will be spaced properly, and if not, it will end at the altitude.

Perhaps it's better to build the string as described and run `.trim()` on it?